<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="converterGroupBox">
         <property name="title">
          <string>Markdown Converter</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="converterLabel">
            <property name="text">
             <string>Converter &amp;Library:</string>
            </property>
            <property name="buddy">
             <cstring>converterComboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="converterComboBox">
            <item>
             <property name="text">
              <string extracomment="Name of a Markdown to HTML converter library">Discount (Default)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Name of a Markdown to HTML converter library">Hoedown</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reveal.js (Presentation)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>213</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editorTab">
      <attribute name="title">
       <string>&amp;Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="fontGroupBox">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="familyLabel">
              <property name="text">
               <string>&amp;Family:</string>
              </property>
              <property name="buddy">
               <cstring>fontComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>6</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="sizeLabel">
              <property name="text">
               <string>Si&amp;ze:</string>
              </property>
              <property name="buddy">
               <cstring>sizeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sizeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="sourceSingleSizedCheckBox">
            <property name="text">
             <string>Markdown Source at Single Size</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tabsGroupBox">
         <property name="title">
          <string>Tabs</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="tabWidthLabel">
            <property name="text">
             <string>&amp;Tab width:</string>
            </property>
            <property name="buddy">
             <cstring>tabWidthSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tabWidthSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="spacesForTabsCheckBox">
            <property name="text">
             <string>Insert Spaces for Tabs</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="otherGroupBox">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="rulerEnableCheckBox">
              <property name="text">
               <string>Display right margin at column:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="rulerPosSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="lineColumnCheckBox">
            <property name="text">
             <string>Show cursor position in status bar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>146</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="htmlPreviewTab">
      <attribute name="title">
       <string>&amp;HTML Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="previewFontsGroupBox">
         <property name="title">
          <string>Fonts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="5">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="sansSerifFontLabel">
            <property name="text">
             <string>S&amp;ans Serif Font:</string>
            </property>
            <property name="buddy">
             <cstring>sansSerifFontComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFontComboBox" name="sansSerifFontComboBox"/>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="standardFontComboBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="standardFontLabel">
            <property name="text">
             <string>S&amp;tandard Font:</string>
            </property>
            <property name="buddy">
             <cstring>standardFontComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFontComboBox" name="serifFontComboBox"/>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="defaultFixedSizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="serifFontLabel">
            <property name="text">
             <string>Se&amp;rif Font:</string>
            </property>
            <property name="buddy">
             <cstring>serifFontComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QComboBox" name="defaultFixedSizeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="defaultSizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="defaultSizeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QFontComboBox" name="fixedFontComboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="fixedFontLabel">
            <property name="text">
             <string>&amp;Fixed Font:</string>
            </property>
            <property name="buddy">
             <cstring>fixedFontComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mathGroupBox">
         <property name="title">
          <string>Math Support</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="mathSupportCheckBox">
            <property name="text">
             <string>Enable Math Support</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mathInlineCheckBox">
            <property name="text">
             <string>Enable LaTeX syntax for inline Math</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="internetTab">
      <attribute name="title">
       <string>&amp;Internet</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="proxyGroupBox">
         <property name="title">
          <string>Configure Proxy to Access the Internet</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="noProxyRadioButton">
            <property name="text">
             <string>No prox&amp;y</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="systemProxyRadioButton">
            <property name="text">
             <string>&amp;Use system proxy settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="manualProxyRadioButton">
            <property name="text">
             <string>&amp;Manual proxy configuration:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="hostLabel">
               <property name="text">
                <string>H&amp;ost:</string>
               </property>
               <property name="buddy">
                <cstring>hostLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="portLabel">
               <property name="text">
                <string>&amp;Port:</string>
               </property>
               <property name="buddy">
                <cstring>portLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="userNameLabel">
               <property name="text">
                <string>User &amp;Name:</string>
               </property>
               <property name="buddy">
                <cstring>userNameLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="passwordLabel">
               <property name="text">
                <string>Pass&amp;word:</string>
               </property>
               <property name="buddy">
                <cstring>passwordLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="hostLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="portLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="userNameLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="passwordLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="snippetsTab">
      <attribute name="title">
       <string>&amp;Snippets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="snippetTableView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>22</number>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="addSnippetButton">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeSnippetButton">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="snippetTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shortcutsTab">
      <attribute name="title">
       <string>Short&amp;cuts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTableWidget" name="shortcutsTable">
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Action</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shortcut</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualProxyRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>OptionsDialog</receiver>
   <slot>manualProxyRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSnippetButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>addSnippetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSnippetButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>removeSnippetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rulerEnableCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rulerPosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSnippetButtonClicked()</slot>
  <slot>removeSnippetButtonClicked()</slot>
 </slots>
</ui>
